main.py

```diff
@@ -0,0 +1,28 @@
+def menu():
+    """This function displays the menu and returns whether the user chose to play or not.
+
+    :return: True if the user chose to play, False otherwise."""
+    while True:
+        print("MENU \n1. New Game \n2. Exit")
+        try:
+            choice = int(input())
+            if choice == 1:
+                return True
+            elif choice == 2:
+                return False
+            else:
+                print("Invalid choice, please try again.")
+                continue
+        except ValueError:
```

Comment:
good job. 
it can still be improved however. generally you want to minimize the size of the block of code that is indented under the try block. right now only line 8 has the ability to raise an exception, not the rest of it. but I will leave that up to you.